CRVResponse/standalone/wls/src/WLSEventAction.cc

```diff
@@ -333,9 +337,9 @@ void WLSEventAction::Draw(const G4Event* evt)
   double digitizationInterval2 = 1.0; //ns
   double noise = 4.0e-4;
   double pedestal = 100; //ADC
-  double calibrationFactor = 394.6; //ADC*ns/PE
+  double calibrationFactor = 391.2; //ADC*ns/PE
   double calibrationFactorPulseHeight = 11.4; //ADC/PE
-  makeCrvWaveform.LoadSinglePEWaveform(_singlePEWaveformFilename.c_str(), 0.5, 1.047, 100, 2.652e-13);
+  makeCrvWaveform.LoadSinglePEWaveform(_singlePEWaveformFilename.c_str(), 0.5, 1.040, 100, 2.652e-13);
```

Comment:
What are these numbers?  They should come either from fcl or proditions.  At least mark these with a comment for a future fix.

Reply:
This code is inside the standalone program, which does not use fcl or any other Offline infrastructure. It is simply used to test/tune the scintillator constants. However, I agree that it should be organized in a better way. I marked it with a FIXME.